java/apps/recovery-portal/src/main/webapp/includes/footer.jsp

```diff
@@ -17,4 +17,107 @@
 --%>
 
 <script src="libs/jquery_3.6.0/jquery-3.6.0.min.js"></script>
-<script src="libs/themes/default/semantic.min.js"></script>
+<script src="libs/themes/asgardio/semantic.min.js"></script>
```

Comment:
Hope the theme name is updated to wso2is.

Reply:
Already changed https://github.com/wso2/identity-apps/blob/86d6527d2e33193ca09d0cc086c28a4d10eb4528/identity-apps-core/apps/recovery-portal/src/main/webapp/includes/footer.jsp#L20